cli/standalone.rs

```diff
@@ -61,6 +77,23 @@ pub fn try_run_standalone_binary(args: Vec<String>) -> Result<(), AnyError> {
   }
 }
 
+fn u64_from_bytes(arr: &[u8]) -> Result<u64, AnyError> {
+  let fixed_arr: &[u8; 8] = arr.try_into()?;
```

Comment:
It'd be useful for debugging to add .context() before ? with description what went wrong